build.gradle

```diff
@@ -68,7 +68,3 @@ subprojects {
 tasks.register("clean", Delete).configure {
   delete rootProject.buildDir
 }
-
-tasks.named("wrapper").configure {
-  distributionType = Wrapper.DistributionType.ALL
```

Comment:
Big fan. But should be in a separate PR.

Reply:
k